src/server/core/shared/route-utils.ts

```
@@ -0,0 +1,243 @@
+  };
+}
+
+export function findAlternativePathWithTimeout(
+  graph: Graph<NodeData, EdgeData>,
+  originId: string,
+  destinationId: string,
+  existingPaths: {
+    path: string[];
+    edges: Array<EdgeData & { source: string; target: string }>;
+  }[],
+  similarityThreshold: number = 0.6
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)